src/api/catalogueItem.test.tsx

```diff
@@ -183,4 +185,134 @@ describe('catalogue items api functions', () => {
       'sends axios request to edit a catalogue item and throws an appropriate error on failure'
     );
   });
+
+  describe('useMoveToCatalogueItem', () => {
+    let props: MoveToCatalogueItem;
+
+    beforeEach(() => {
+      props = {
+        selectedItems: [
+          {
+            catalogue_category_id: '657305a01e468454e97b6389',
+            manufacturer: {
```

Comment:
The manufacturer should be `manufacturer_id` and an Id now that #144 has been merged